app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -11,13 +11,19 @@ enum class EventCategory {
   COMMUNITY
 }
 
+data class Ticket(val name: String, val price: Double, val quantity: Int)
+// new event struct final
 data class Event(
-    val id: String,
-    val hostUserId: String,
-    val name: String,
+    val eventName: String,
+    val eventPhoto: String,
```

Comment:
nit: what does that String contains about the image ? a comment would be nice !

Reply:
string is reference to image path